### DISCUSSION TITLE
Input > $size doesn't do what the docs say it does

### DISCUSSION THREAD

**Discussion by mitya33** [buefy/buefy#3658] [Q&A]
The docs [say](https://buefy.org/documentation/input) that the `size` prop should set the vertical size of the input.

At least for me, on a textarea, this isn't the case - it merely sets the font size of it. Usage:

`<b-input v-model='bio' type='textarea' maxlength='10000' size='is-large' />`

Anyone else get this?

**Comment by GMartigny** [comment]
[I don't reproduce](https://codepen.io/GMartigny/pen/gOvXRoP?editors=1000).
